explainer-server/app/views/explainEditor.scala.html

```diff
@@ -25,5 +28,18 @@
     </header>
 }
 @templates.main("Explainer Editor", header, toolbar){
+
+    <script type="text/javascript">
+        /*
+         * This function was put here at opposition to within the scala code because
+         * the read operation of the text area content uses cached values in the JS code generated by ScalaJS.
+         */
+        function updateWordCountDisplay(){
+            var contents = $.trim($("#explainer-editor__body-wrapper__input").val())

```

Comment:
in general it's better avoiding ids.

ids are not future compatible if you ever want to change the UI and have two explainer in a page